jobsapp/api/views/common.py

```diff
@@ -27,3 +34,29 @@ def get_queryset(self) -> "QuerySet[Job]":
             )
         else:
             return self.serializer_class.Meta.model.objects.filter(filled=False)
+
+
+@api_view(["POST"])
```

Comment:
This is not wrong, but since we are using django rest framework you can use the APIViews, GenericViews or Viewsets. Creating a class with a method `post` or `create` depending on what class you will use will be nicer and more consistent with the rest of the views that we have.

Reply:
Yep, makes sense.